packages/migrate/src/commands/DbPull.ts

```diff
@@ -456,23 +456,9 @@ ${`Run ${chalk.green(getCommandWithExecutor('prisma generate'))} to generate Pri
             .join('\n')
         } else if (warning.code === 4) {
           message += warning.affected.map((it) => `- Enum "${it.enm}", value: "${it.value}"`).join('\n')
-        } else if (
-          warning.code === 5 ||
-          warning.code === 6 ||
-          warning.code === 8 ||
-          warning.code === 11 ||
-          warning.code === 12 ||
-          warning.code === 13 ||
-          warning.code === 16
-        ) {
+        } else if (warning.code === 5 || warning.code === 6 || warning.code === 8) {
           message += warning.affected.map((it) => `- Model "${it.model}", field: "${it.field}"`).join('\n')
-        } else if (
-          warning.code === 7 ||
-          warning.code === 14 ||
-          warning.code === 15 ||
-          warning.code === 18 ||
-          warning.code === 19
-        ) {
+        } else if (warning.code === 7 || warning.code === 14 || warning.code === 18 || warning.code === 19) {
```

Comment:
```suggestion
        } else if [7, 14, 18, 19].includes(warning.code) {
```

Reply:
Unfortunately, that would mean that the code is not type safe anymore with this suggestion

Reply:
See before
<img width="688" alt="Screenshot 2023-01-27 at 17 15 07" src="https://user-images.githubusercontent.com/1328733/215136342-f33c0fda-0fb9-46b9-b292-acadc4cf1778.png">
and with your suggestion
<img width="576" alt="Screenshot 2023-01-27 at 17 14 45" src="https://user-images.githubusercontent.com/1328733/215136345-f7ad6d98-ebe8-4f73-9ac1-818da879e752.png">